cabal.project

```diff
@@ -128,58 +132,61 @@ source-repository-package
 source-repository-package
-  type: git
-  location: https://github.com/input-output-hk/ouroboros-network
-  tag: c764553561bed8978d2c6753d1608dc65449617a
-  --sha256: 0hdh7xdrvxw943r6qr0xr4kwszindh5mnsn1lww6qdnxnmn7wcsc
-  subdir:
-    monoidal-synchronisation
-    network-mux
-    ouroboros-consensus
-    ouroboros-consensus-byron
-    ouroboros-consensus-cardano
-    ouroboros-consensus-protocol
-    ouroboros-consensus-shelley
-    ouroboros-network
-    ouroboros-network-framework
-    ouroboros-network-testing
-
+   type: git
+   location: https://github.com/input-output-hk/cardano-ledger
+   tag: cb29b038a5e8ae5a91f3e6fbc34b474c27cc1d87
+   --sha256: 0gxb9xghsacr0qaljhq3qzvmb2f1wwmly89prwqy4fdm9mvqiq95
+   subdir:
+     eras/alonzo/impl
+     eras/alonzo/test-suite
+     eras/babbage/impl
+     eras/babbage/test-suite
+     eras/byron/chain/executable-spec
+     eras/byron/crypto
+     eras/byron/crypto/test
+     eras/byron/ledger/executable-spec
+     eras/byron/ledger/impl
+     eras/byron/ledger/impl/test
+     eras/conway/impl
+     eras/shelley/impl
+     eras/shelley/test-suite
+     eras/shelley-ma/impl
+     eras/shelley-ma/test-suite
+     libs/cardano-ledger-core
+     libs/cardano-ledger-pretty
+     libs/cardano-protocol-tpraos
+     libs/cardano-data
+     libs/vector-map
+     libs/set-algebra
+     libs/small-steps
+     libs/small-steps-test
+     libs/non-integral
+
+-- 774f3e445e4cadf78d38a2b9d20b620d3b576835 is c764553561bed8978d2c6753d1608dc65449617a+44
+-- a38f898a3740925ccaa0617fa870ca7054a485a6 is 774f3e445e4cadf78d38a2b9d20b620d3b576835+1 (strict-containers -> cardano-strict-containers)
+-- And has the adjustments for the ledger refactor
 source-repository-package
-  type: git
-  location: https://github.com/input-output-hk/cardano-ledger
-  tag: f49879a79098d9372d63baa13b94a941a56eda34
-  --sha256: 0i9x66yqkrvx2w79dy6lzlya82yxc8567rgjj828vc2d46d6nvx6
-  subdir:
-    eras/alonzo/impl
-    eras/alonzo/test-suite
-    eras/babbage/impl
-    eras/babbage/test-suite
-    eras/byron/chain/executable-spec
-    eras/byron/crypto
-    eras/byron/crypto/test
-    eras/byron/ledger/executable-spec
-    eras/byron/ledger/impl
-    eras/byron/ledger/impl/test
-    eras/shelley/impl
-    eras/shelley/test-suite
-    eras/shelley-ma/impl
-    eras/shelley-ma/test-suite
-    libs/cardano-ledger-core
-    libs/cardano-ledger-pretty
-    libs/cardano-protocol-tpraos
-    libs/cardano-data
-    libs/vector-map
-    libs/set-algebra
-    libs/small-steps
-    libs/small-steps-test
-    libs/non-integral
-
-source-repository-package
-  type: git
-  location: https://github.com/input-output-hk/cardano-prelude
-  tag: 6ea36cf2247ac0bc33e08c327abec34dfd05bd99
-  --sha256: 0z2y3wzppc12bpn9bl48776ms3nszw8j58xfsdxf97nzjgrmd62g
-  subdir:
-    cardano-prelude
-    cardano-prelude-test
+   type: git
+   location: https://github.com/input-output-hk/ouroboros-network
+   tag: a38f898a3740925ccaa0617fa870ca7054a485a6
```

Comment:
@disassembler note that we must not make release from this tag, because it would move backward from what will be released in `1.35.5`.  It must be at least from input-output-hk/ouroboros-network@9f91e639b and preferably from input-output-hk/ouroboros-network@9f91e639b.

Reply:
Shall I bump it to `b83d44e` in this PR?

Reply:
Actually, LGTM and then I'll open a new PR

Reply:
It would be cleaner to have this in the same PR, and also use `CHaP`, but for that you'd need to wait until we publish the new version of `ouroboros-network` to `CHaP`.

We need to speak with the `consensus` team if one of these commits is ok for them, or just let them review the PR to `cardano-haskell-packages` repo.

Reply:
I'm in favour of making sure we have a ouroboros-network release for this in CHaP (and ekg-forward). I'm happy to help with any such endeavour.